UITests/UITests.App/MainPage.xaml

```diff
@@ -9,11 +9,17 @@
     Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
 
     <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="*" />
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
         <testhelpers:TestAutomationHelpersPanel />
 
-        <StackPanel HorizontalAlignment="Center" VerticalAlignment="Center">
-            <Button Content="Click Me" Click="Button_Click"/>
-            <TextBlock x:Name="textBlock"/>
+        <Frame x:Name="navigationFrame" />
+        
+        <StackPanel Orientation="Horizontal" Grid.Row="1" >
```

Comment:
However, I don't like the use of the buttons here to control the page navigation, we should build this into the infrastructure itself. So the MainPage has a function of understanding how to switch tasks and can evaluate a string and navigate to that page based on reflection or something, similar to how we navigate within the SampleApp currently:

https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/49760c325ecbddab254b61214e5a808e866e51d6/Microsoft.Toolkit.Uwp.SampleApp/Models/Sample.cs#L100